assets/xml/objects/object_tro.xml

```diff
@@ -9,6 +9,9 @@
         <Texture Name="object_tro_Tex_000F80" OutName="tex_000F80" Format="rgba16" Width="16" Height="8" Offset="0xF80" />
         <Texture Name="object_tro_Tex_001080" OutName="tex_001080" Format="rgba16" Width="8" Height="8" Offset="0x1080" />
         <Texture Name="object_tro_Tex_001100" OutName="tex_001100" Format="ci8" Width="8" Height="8" Offset="0x1100" />
-        <!-- <Blob Name="object_tro_Blob_001140" Size="0x1820" Offset="0x1140" /> -->
+        <Texture Name="object_tro_Tex_001140" OutName="tex_001140" Format="rgba16" Width="32" Height="32" Offset="0x1140"/> <!-- eye open -->
+        <Texture Name="object_tro_Tex_001940" OutName="tex_001940" Format="rgba16" Width="32" Height="32" Offset="0x1940"/> <!-- eye half open-->
+        <Texture Name="object_tro_Tex_002140" OutName="tex_002140" Format="rgba16" Width="32" Height="32" Offset="0x2140"/> <!-- eye closed-->
+        <Skeleton Name="object_tro_Skel_002950" Type="Normal" LimbType="Standard" Offset="0x2950" />
```

Comment:
```suggestion
        <Limb Name="object_tro_Standardlimb_002940" Type="Standard" EnumName="OBJECT_TRO_LIMB_1" Offset="0x2940" />
        <Skeleton Name="object_tro_Skel_002950" Type="Normal" LimbType="Standard" LimbNone="OBJECT_TRO_LIMB_NONE" LimbMax="OBJECT_TRO_LIMB_MAX" EnumName="ObjectTroLimb" Offset="0x2950" />
```
Really nice work with the xml, the only thing left I say would be to add the limb, and add generic enum information for the time being.